tools/hexmerge.py

```diff
@@ -35,7 +35,7 @@
 
 """Merge content of several hex files into one file."""
 
-VERSION = '2.2'
+VERSION = '2.3.0'
```

Comment:
Why bump the version number here?

Reply:
Actually I realized why... but drop this change anyway! Better to replace tools/hexmerge.py with a symlink so that's what I've just done.